src/Utility/Privacy.js

```diff
@@ -37,7 +37,11 @@ export default class Privacy {
     }
 
     static setAllowed(privacy_action, value) {
-        Cookie.set(this.cookieNameForAction(privacy_action), value, { expires: 365 });
+        if(value !== privacy_action.default) Cookie.set(this.cookieNameForAction(privacy_action), value, { expires: 365 });
+        else {
+            let cookie_value = Cookie.get(this.cookieNameForAction(privacy_action));
+            if(cookie_value!==undefined) Cookie.remove(this.cookieNameForAction(privacy_action));
```

Comment:
Are you sure that this is necessary? I am pretty sure that you can just delete the cookie, even if it doesn't exist.

Reply:
Oh, didn't think about that, I'll make the necessary change